docs/01-getting-started.md

```diff
@@ -6,14 +6,23 @@ a Sleeper instance with a simple schema, and writes some random data into a tabl
 scripts to see how much data is in the system, run some example queries, and view logs to help understand what the system is
 doing. It is best to do this from an EC2 instance as a significant amount of code needs to be uploaded to AWS.
 
-Before running this demo functionality, you will need the following intalled (see the [deployment guide](02-deployment-guide)
+Before running this demo functionality, you will need the following intalled (see the [deployment guide](02-deployment-guide.md)
 for more information on getting set up correctly) and you will need your CLI to be logged into your AWS account:
 
-- Maven
-- The AWS CLI
-- CDK
-- Docker
-- Java
+* [AWS CDK](https://docs.aws.amazon.com/cdk/latest/guide/cli.html)
+* [AWS CLI](https://docs.aws.amazon.com/cli/latest/userguide/install-cliv2.html): v2
+* [Bash](https://www.gnu.org/software/bash/): At least version 4: use `bash --version`.  The default version on a Mac is 3.2, which is too old.
+* [Docker](https://docs.docker.com/get-docker/)
+* [Java 8](https://openjdk.java.net/install/)
+* [Maven](https://maven.apache.org/): Tested with 3.8.6.
+* [NodeJS / NPM](https://github.com/nvm-sh/nvm#installing-and-updating)
+
+First run (note this command cannot be run from the `sleeper` directory directly as the Java CDK classes are not yet compiled into the JAR named in `cdk.json`):
```

Comment:
Can we update this to note that this only needs doing if cdk has not previously been bootstrapped in your account? And also note that instructions for this are in the "Bootstrapping CDK" section of 02-deployment-guide.md? We also recently removed the instructions for bootstrapping CDK from 02-deployment-guide.md in favour of just including the URL to AWS's instructions for bootstrapping it, so I'd prefer not to include the commands below.

Reply:
Added a note to say this is a one time thing and agree that the instructions were clear in 02-deployment-guide.md.  I hadn't ready that until after I'd attempted an installation though (my fault).